### DISCUSSION TITLE
If I want to use my own dataset, the format needs to be consistent with Mvtec, because I didn't find the relevant dataloader

### DISCUSSION THREAD

**Discussion by wangk98** [open-edge-platform/anomalib#287] [Q&A] [answered]
**Describe the bug**
- A clear and concise description of what the bug is.

**To Reproduce**

**Expected behavior**
- A clear and concise description of what you expected to happen.

**Screenshots**
- If applicable, add screenshots to help explain your problem.

**Hardware and Software Configuration**
- OS: [Ubuntu, OD]
- NVIDIA Driver Version [470.57.02]
- CUDA Version [e.g. 11.4]
- CUDNN Version [e.g. v11.4.120]
- OpenVINO Version [Optional e.g. v2021.4.2]

**Additional context**
- Add any other context about the problem here.

**Comment by samet-akcay** [comment, marked as answer]
@wangk98, you could use the [folder dataset](https://github.com/openvinotoolkit/anomalib/blob/development/anomalib/data/folder.py#L304), for which you could use the following as a [sample config](https://github.com/openvinotoolkit/anomalib#custom-dataset).

**Reply by wangk98** [reply]
Sorry to bother you again, I got the following error when I experimented with padim, the dataset has been changed to the same format as MVTec
![image](https://user-images.githubusercontent.com/74230385/166179095-865cfa85-72b1-412b-a49f-ca737243578a.png)

**Reply by samet-akcay** [reply]
@wangk98, sorry, this just slipped our attention, did you manage to solve this?
